packages/custodyController/src/types.ts

```diff
@@ -5,7 +5,6 @@ export interface CustodyAccountDetails {
   name: string;
   custodianDetails: IEthereumAccountCustodianDetails;
   labels: string[];
-  apiUrl: string;
   custodyType: string;
   chainId: number;
   custodianName: string; // e.g. saturn-dev - this will be used to look up custodian details from the MMI configuration store
```

Comment:
Renaming `custodianName` to `envName` could be tricky here. As `state.custodyAccountDetails` is used extensively and it relies on that name.

Probably best to do it as a follow up.